src/actions/index.js

```diff
@@ -469,6 +471,19 @@ function updateOutput(
         });
     }
 
+    if(hooks.request_pre !== null) {
+        if(typeof hooks.request_pre === 'function') {
```

Comment:
For consistency, I try to use `ramda` throughout this library. So, this would be `if (R.type(hooks.request_pre) === 'function')